core/src/test/java/org/kiwix/kiwixmobile/core/bookmark/viewmodel/BookmarkTestHelpers.kt

```diff
@@ -0,0 +1,35 @@
+package org.kiwix.kiwixmobile.core.bookmark.viewmodel
+
+import org.kiwix.kiwixmobile.core.bookmark.adapter.BookmarkItem
+
+// dateFormat = d MMM yyyy
+//             5 Jul 2020
+fun bookmark(
+  bookmarkTitle: String = "bookmarkTitle",
+  isSelected: Boolean = false,
+  id: Long = 2
+): BookmarkItem {
+  return BookmarkItem(
```

Comment:
perhaps you don't need it for the tests but the style is to add a default parameter for every constructor param